analytics/src/test/java/com/segment/analytics/LoggerTest.kt

```diff
@@ -0,0 +1,128 @@
+package com.segment.analytics
+
+import android.util.Log
+import com.segment.analytics.integrations.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.robolectric.shadows.ShadowLog
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class LoggerTest {
+    @Test
+    fun verboseLevelLogsEverything() {
+        val logger = Logger.with(Analytics.LogLevel.VERBOSE)
+
+        logger.debug("foo")
+        logger.info("bar")
+        logger.verbose("qaz")
+        logger.error(null, "qux")
+
+        Assertions.assertThat(ShadowLog.getLogs()).hasSize(4)
+    }
+
+    @Test
+    fun verboseMessagesShowInLog() {
+        val logger = Logger.with(Analytics.LogLevel.VERBOSE)
+        logger.verbose("some message with an %s", "argument")
+        Assertions.assertThat(ShadowLog.getLogs())
+                .containsExactly(
+                       LogItemBuilder()
+                                .type(Log.VERBOSE)
+                                .msg("some message with an argument")
+                                .build())
+    }
+
+    @Test
+    fun debugMessagesShowInLog(){
```

Comment:
Nitpick: Try and keep your style and spacing consistent. In this case there should be a space between ) and {

`fun debugMessagesShowInLog() {`